pool/app/contact/contact.mli

```diff
@@ -160,6 +164,12 @@ module Preview : sig
   val email_address : t -> Pool_user.EmailAddress.t
 end
 
+val searchable_by : (Pool_common.Message.Field.t * string) list
```

Comment:
Check the API, currently `_by`, `_columns` and `fields` is exposed. The full information is in the `_by` variable, the others just return the map first or second of it.

It's also not possible to extend the `_by` variable to get the extended `_columns` because it always refers to the base list of it.

Which of these functions are needed?
Or only expose the `_by` function and use the `CCList.map fst` within the repository or the places where you actually need them. With this the corresponding key/value pair is always delivered together.